services/azure-devops/azure-devops-tests.service.js

```diff
@@ -65,48 +53,84 @@ export default class AzureDevOpsTests extends AzureDevOpsBase {
     queryParamSchema: testResultQueryParamSchema,
   }
 
-  static examples = [
-    {
-      title: 'Azure DevOps tests',
-      staticPreview: this.render({
-        passed: 20,
-        failed: 1,
-        skipped: 1,
-        total: 22,
-      }),
-      ...commonAttrs,
-    },
-    {
-      title: 'Azure DevOps tests (compact)',
-      staticPreview: this.render({
-        passed: 20,
-        failed: 1,
-        skipped: 1,
-        total: 22,
-        isCompact: true,
-      }),
-      ...commonAttrs,
+  static openApi = {
+    '/azure-devops/tests/{organization}/{project}/{definitionId}': {
+      get: {
+        summary: 'Azure DevOps tests',
+        description,
+        parameters: [
+          pathParam({
+            name: 'organization',
+            example: 'azuredevops-powershell',
+          }),
+          pathParam({
+            name: 'project',
+            example: 'azuredevops-powershell',
+          }),
+          pathParam({
+            name: 'definitionId',
+            example: '1',
+          }),
+          queryParam({
+            name: 'passed_label',
+            example: 'passed',
+          }),
+          queryParam({
+            name: 'failed_label',
+            example: 'failed',
+          }),
+          queryParam({
+            name: 'skipped_label',
+            example: 'skipped',
+          }),
+          queryParam({
+            name: 'compact_message',
+            example: null,
+          }),
```

Comment:
instead of defining these common params here (twice), you can import `testResultOpenApiQueryParams` from `test-results.js` and use `...testResultOpenApiQueryParams` here.

Reply:
Good idea!